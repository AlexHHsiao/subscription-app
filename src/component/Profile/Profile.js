import React from "react";

const Profile = () => (
    <div>
        Profile setting here!
    </div>
);

export default Profile;
