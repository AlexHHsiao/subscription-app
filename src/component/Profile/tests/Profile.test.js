import React from 'react';
import {render} from '@testing-library/react';
import Profile from '../Profile';

test('<Profile/>', () => {
    const {container: {firstChild}} = render(<Profile/>);
    expect(firstChild).toMatchSnapshot();
});
