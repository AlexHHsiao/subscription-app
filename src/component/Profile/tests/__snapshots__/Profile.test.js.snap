// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Profile/> 1`] = `
<div>
  Profile setting here!
</div>
`;
